<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form3</class>
 <widget class="QWidget" name="Form3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ustawienia</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="enabledB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Włącz wtyczkę</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="extsB"/>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="notReadID3">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Nie odczytuj ID3 dla wpisów playlisty</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="searchPartsB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Kiedy plik MP3 nie jest kompletny i brakuje mu jakiś części, to szuka dane aż do końca pliku</string>
     </property>
     <property name="text">
      <string>Wyszukuj części</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="wait4DataB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Kiedy połączenie z serwerem zostanie zerwane lub się zawiesi, QMPlay nie przerwie odtwarzania tylko będzie czekał na dane</string>
     </property>
     <property name="text">
      <string>Czekaj na dane</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="ApplyB">
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Zastosuj</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>enabledB</tabstop>
  <tabstop>extsB</tabstop>
  <tabstop>notReadID3</tabstop>
  <tabstop>searchPartsB</tabstop>
  <tabstop>wait4DataB</tabstop>
  <tabstop>ApplyB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
